src/blocks/HeaderBlockBrandPivot/HeaderTop.tsx

```diff
@@ -0,0 +1,279 @@
+  }
+`
+
+const ContactLink = styled.a`
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+
+  ${TABLET_BP_UP} {
+    margin-right: 4rem;
+  }
+
+  svg {
+    margin-right: 0.5rem;
+  }
+`
+
+const MarketPicker = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+`
+
+const MarketPickerToggle = styled.button`
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  border: none;
+  color: inherit;
+  background: transparent;
+  letter-spacing: 0.5px;
+  cursor: pointer;
+
+  &:focus {
+    outline: 0;
+  }
+
+  &:after {
+    display: inline-block;
+    content: '';
+    margin-top: 1px;
+    margin-left: 0.5rem;
+    border-width: 6px 6px 0 6px;
+    border-color: currentColor transparent transparent transparent;
+    border-style: solid;
+  }
+
+  svg {
+    margin-right: 0.5rem;
+  }
+`
+
+const MarketPickerWrapper = styled.div<{ isOpen: boolean; inverse: boolean }>`
+  display: ${(props) => (props.isOpen ? 'block' : 'none')};
+  position: absolute;
+  top: 2.5rem;
+  left: 0;
+  width: 100vw;
+  max-width: 270px;
+  z-index: 103;
+  background-color: ${(props) =>
+    props.inverse ? colorsV3.gray100 : colorsV3.gray700};
+  color: ${(props) => (props.inverse ? colorsV3.gray900 : colorsV3.gray100)};
+  border-radius: 0.5rem;
+
+  ${TABLET_BP_UP} {
+    left: auto;
+    right: 0;
+    max-width: 300px;
+  }
+`
+
+const MarketList = styled.ul`
+  display: block;
+  margin: 0;
+  padding: 2.5rem 1.5rem;
+`
+
+const MarketItem = styled.li`
+  display: block;
+  margin-bottom: 2rem;
+  color: inherit;
+
+  &:last-of-type {
+    margin-bottom: 0;
+  }
+`
+
+const MarketPrimaryLink = styled.a`
+  display: flex;
+  align-items: center;
+  margin-bottom: 1rem;
+  text-decoration: none;
+`
+
+const MarketTitle = styled.span`
+  display: block;
+  margin-left: 1rem;
+  font-size: 1.25rem;
+
+  :hover {
+    opacity: 0.7;
+  }
+`
+
+const MarketLocales = styled.ul`
+  display: flex;
+  margin: 0 0 0 2.5rem;
+  padding: 0;
+`
+
+const MarketLocale = styled.li`
+  display: flex;
+`
+
+const MarketLink = styled.a`
+  padding: 0 1rem;
+  color: inherit;
+  text-decoration: none;
+
+  :hover {
+    opacity: 0.7;
+  }
+`
+
+const Divider = styled.span`
+  display: block;
+  content: '';
+  width: 1px;
+  height: 1.25rem;
+  background-color: currentColor;
+`
+
+const CloseButton = styled('button')`
+  position: absolute;
+  top: 1rem;
+  right: 0.75rem;
+  display: flex;
+  width: 1.25rem;
+  height: 1.25rem;
+  padding: 0;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: currentColor;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+
+  :focus {
+    outline: none;
+  }
+
+  :hover {
+    opacity: 0.8;
+  }
+
+  svg {
+    transform: rotate(45deg);
+  }
+`
+
+export const HeaderTop: React.FC<HeaderTopProps> = ({
+  transparent,
+  inverse,
+}) => {
+  const [openMarketPicker, toggleMarketPicker] = useState<boolean>(false)
```

Comment:
this sounds like a function that'll open the market picker, maybe `isMarketPickerOpen` or `marketPickerOpen`?